### PR TITLE
List Users Following tags

### PR DESCRIPTION
**Pull request by StlMaris123**
#1421 

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by StlMaris123** [comment]
I have done something on:
`Node has X tags`
`(revised) Get all users who are following the new tag`

Please shed some light on 
`For each of these following users, none of the existing tags is being followed by the user`
`the user has not already gotten an email for that node creation or any of its tag creations`

**Comment by StlMaris123** [comment]
What could be the issue causing Travis_ci to fail?

**Comment by jywarren** [comment]
hi @publiclab/soc - can anyone look through the logs here and help @StlMaris123 debug? Thanks!

**Comment by ryzokuken** [comment]
@StlMaris123 There's a pending migration. Maybe you could try migrating the database locally by running `rake db:migrate`? That  *should* solve the issue.

Great work, by the way.

**Comment by jywarren** [comment]
Ah, if there's a pending migration you need to update the schema file, i think?

File: db/migrate/20170612162455_add_email_sent_at_to_rusers.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddEmailSentAtToRusers < ActiveRecord::Migration
```
Comment (jywarren) [inline db/migrate/20170612162455_add_email_sent_at_to_rusers.rb]: But I think we want to separate out this migration into its own PR, right? because this PR would be just for the model method `followers_who_dont...` -- right? If you can move this migration to its own branch, then remove it from this PR then things should pass. Good catch, @ryzokuken!

Comment (ryzokuken) [inline db/migrate/20170612162455_add_email_sent_at_to_rusers.rb]: Glad to be of help

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+class UserTest < ActiveSupport::TestCase
```
Comment (jywarren) [inline test/models/user_test.rb]: Also, here, I don't see an explicit test of `followers_who_dont_follow_tags` -- but is this still in progress? Great to see the test shaping up here!

**Comment by ryzokuken** [comment]
@StlMaris123 again, look's like you need to migrate.

**Comment by StlMaris123** [comment]
Splitting this into two PR's:
- return a list of users who follow this tag, but don't follow the list of given tag
- A check that user has not received any email
